attack-theme/templates/general/base-template.html

```diff
@@ -116,6 +116,8 @@
     <script src="/theme/scripts/bootstrap.bundle.min.js"></script>
     <script src="/theme/scripts/site.js"></script>
     <script src="/theme/scripts/flexsearch.es5.js"></script> 
+    <script src="/theme/scripts/localforage.min.js"></script>
+    <script src="/theme/scripts/settings.js?{{ range(1, 9002) | random }}"></script>
```

Comment:
This change (the addition of the random ID to settings.js) seems to cause an issue with the link checker:
```
/index.html
	- /theme/scripts/settings.js?3477/index.html
/404.html
	- /theme/scripts/settings.js?43/index.html
```
(which goes on for every page in the site). Probably will require an update to the link checker code so it doesn't try to append the index.html suffix.